roles/openshift_daemonset_config/templates/daemonset.yml.j2

```diff
@@ -150,3 +149,7 @@ spec:
       - name: host
         hostPath:
           path: /
+      - hostPath:
+          path: /run/docker.sock
```

Comment:
I expected `/var/run/docker.sock` here

Reply:
I did as well but /var/run is a symlink to /run.  

That's why this works.  I'm fine either way.